src/celpy/c7nlib.py

```diff
@@ -52,8 +52,8 @@ def process(self,
             for resource in resources:
                 with C7NContext(filter=the_filter):
                     cel_activation = {
-                        "resource": celpy.json_to_cel(resource),
-                        "now": celpy.celtypes.TimestampType(now),
+                        "Resource": celpy.json_to_cel(resource),
```

Comment:
Need to update these `Resource`, `Now`, and `Event` fields throughout the codebase to be `resource`, `now`, and `event`